exercises/binary-search/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile "junit:junit:4.12"
+  testCompile "org.assertj:assertj-core:3.2.0"
```

Comment:
Hello, also agree. Actually used jUnit assertions in the test. In some other projects AssertJ is used, so It looks like a standard for the project. Removed the dependency.